src/test/java/org/mule/modules/watsonalchemylanguage/automation/functional/EntitiesTestCases.java

```diff
@@ -58,4 +57,19 @@ private void testPrecenseOf(List<Entity> entities, String... entitiesToTest) {
 		assertEquals(String.format("The response doesn't contains all the expect entities: %s", flatEntities),
 				entitiesToTest.length, cont);
 	}
+	
+	private EntitiesRequest buildTestRequest(String source, Integer maxRetrieve) {
+		EntitiesRequest request = new EntitiesRequest();
+		request.setSource(source);
+		request.setMaxRetrieve(maxRetrieve);
+		
+		return request;
+	}
+	
+	private EntitiesRequest buildTestRequest(String source) {
```

Comment:
This is not polymorphism, this is two duplicated methods with only one line of difference.